builder/frameworks/espidf.py

```
@@ -189,177 +191,207 @@ def _get_installed_standard_pip_packages():
-                file_path = join(PROJECT_DIR,file.lstrip("file://").split(os.path.sep)[-1])
-                if os.path.exists(file_path):
-                    with open(file_path, 'r') as file:
-                        target = file.read()
-                else:
-                    print("File not found:", file_path)
-                    return ""
-                return target
+
+    def read_local_config(file_path):
+        """Read configuration from local file."""
+        full_path = join(PROJECT_DIR, file_path.lstrip("file://").split(os.path.sep)[-1])
```

Ruff (0.11.9)

Using `.strip()` with multi-character strings is misleading

(B005)